src/state/queries/post.ts

```diff
@@ -0,0 +1,236 @@
+import {AppBskyFeedDefs} from '@atproto/api'
+import {
+  useQuery,
+  useQueryClient,
+  useMutation,
+  QueryClient,
+} from '@tanstack/react-query'
+import {useSession} from '../session'
+
+export const RQKEY = (postUri: string) => ['post', postUri]
```

Comment:
Yeah same here. If we update cache with a like, then switch users, I imagine the post could in some cases still show as "liked" if we aren't careful. So might want did here? I could be wrong